libs/blocks/z-pattern/z-pattern.js

```diff
@@ -86,5 +86,11 @@ export default function init(el) {
   const mediaItems = el.querySelectorAll(':scope > .media');
```

Comment:
```suggestion
  const mediaItems = el.querySelectorAll(':scope > .media');
  const variants = ['checklist', 'qr-code'];
```
Just a suggestion of how we can maybe use an array to loop these values incase others are needed in the future.

Reply:
Resolved https://github.com/adobecom/milo/pull/971/commits/cc6a21365619e5afcd3e7dc6ac7814b8b4f93072 @ryanmparrish